Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -625,4 +625,9 @@ public ConceptEntry getOriginalConceptEntry(String id) {
 
         return null;
     }
+
```

Comment:
this class still uses the db4o database, rigth? It needs to use the new sql classes so it serves up the migrated data. This is true for all manager classes.